### PR TITLE
tuning: recommend minimum browser specs

### PR DESCRIPTION
**Pull request by moio**
## Description

Clarify that in high scale scenarios additional requirements apply to the machine running the browser that navigates the Rancher UI.

## Comments

This applies to all versions current and future - for the time being I changed all files identically and opened this PR against the main branch.

### CODE REVIEW

**Comment by moio** [comment]
cc @CamrynCarter

File: docs/reference-guides/best-practices/rancher-server/tuning-and-best-practices-for-rancher-at-scale.md [hunk 1]
```diff
@@ -118,3 +118,13 @@ The two main bottlenecks to [etcd performance](https://etcd.io/docs/v3.4/op-guid
...
 
 Etcd performance will also be negatively affected by network latency between nodes as that will slow down network communication. Etcd nodes should be located together with Rancher nodes.
+
+### Browser requirements
+
+At high scale Rancher will transfer more data from the upstream cluster to UI components running in the browser, and those will also need to perform more processing.
+
+For best performance, ensure that the host running the hardware meets these requirements:
+ - 2020 i5 10th generation Intel (4 cores) or equivalent
+ - 8 GB RAM
```
Comment (martyav) [inline docs/reference-guides/best-practices/rancher-server/tuning-and-best-practices-for-rancher-at-scale.md:128]:
```suggestion
 - 8 GB RAM
```
Added non-breaking space as recc'd by Review Dog

File: docs/reference-guides/best-practices/rancher-server/tuning-and-best-practices-for-rancher-at-scale.md [hunk 2]
```diff
@@ -118,3 +118,13 @@ The two main bottlenecks to [etcd performance](https://etcd.io/docs/v3.4/op-guid
...
+ - Total network bandwith to the upstream cluster: 72 Mb/s (equivalent to a single 802.11n Wi-Fi 4 link stream, ~8 MB/s http download throughput)
+ - Round-trip time (ping time) from browser to upstream cluster: 150ms or less
```
Comment (martyav) [inline docs/reference-guides/best-practices/rancher-server/tuning-and-best-practices-for-rancher-at-scale.md:130]:
```suggestion
 - Total network bandwith to the upstream cluster: 72 Mb/s (equivalent to a single 802.11n Wi-Fi 4 link stream, ~8 MB/s http download throughput)
 - Round-trip time (ping time) from browser to upstream cluster: 150 ms or less
```
More spaces

**Review by martyav — CHANGES_REQUESTED** [review]
Just reviewed the file in `/docs`, but suggestions should apply to the others as relevant

**Comment by moio** [comment]
Note I disagree with vale about the ordinal in this case - Intel marketing material and the industry in general uses it ubiquitously to indicate microprocessor architectures. I think spelling out the ordinal would make it less familiar for readers.

eg.
https://www.anandtech.com/show/14782/intel-launches-comet-lakeu-and-comet-lakey-10th-gen-core-for-low-power-laptops

**Comment by martyav** [comment]
@moio It's a good point, I hesitated on that one. The docs team went over how applicable the OpenSUSE Vale style guide is to our situation last week, and made note of some rules that need adjusting. This probably needs to be updated to something like "Ordinals are preferred when listing tech specs"

**Review by martyav — APPROVED** [review]

**Review by martyav — APPROVED** [review]

**Pull request merged**